composer.json

```diff
@@ -21,7 +21,8 @@
         "symfony-cmf/testing": "^1.3@dev",
         "symfony/dependency-injection": "^2.0.5|3.*",
         "symfony/config": "^2.2|3.*",
-        "symfony/event-dispatcher": "^2.1|3.*"
+        "symfony/event-dispatcher": "^2.1|3.*",
+        "friendsofsymfony/jsrouting-bundle": "@dev"
```

Comment:
do we require this or is it some other bundle that requires it? then the `@dev` should go in there rather than here, i think.